api/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Aug 23 08:45:28 EDT 2017
+#Thu May 10 14:42:28 CDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-bin.zip
```

Comment:
What are the gradle changes for?

Reply:
FWIW, intellij likes to make this change for me as well.

Reply:
What @rushtong said. I assume this change is fine and will avoid us having to make it again.